README.md

```diff
@@ -29,7 +29,7 @@ def data_function():
 device_info = DeviceInfo("testDevice", "test device")
 credentials = ClientCredentials("thingoo-device", "CLIENT_SECRET")
 temp = Entity("temp", "SENSOR", "DECIMAL", "C")
-connector = Connector(device_info, "dev.thingoo.xyz", credentials, [temp])
+connector = HTTPConnector(device_info, "dev.thingoo.xyz", credentials, [temp])
```

Comment:
When we already have a ThingooConnector facade, shouldn't we just use it here instead?

Reply:
Ofc it should be but this commit was made before creating ThingooConnector class. I haven't updated the README.md yet.
Moreover, args for ThingooConnector will be different.